app/views/backend/purchases/show.html.haml

```diff
@@ -41,7 +41,7 @@
 
   - if resource.affair
     - c.cobble :affair do
-      = affair_of(resource, default: 'PurchasePayment')
+      = affair_of(resource, default: 'PurchasePayment', purchase_payment_attributes: { responsible_id: resource.responsible_id, paid_at: resource.payment_at, to_bank_at: resource.payment_at})
```

Comment:
Do you mean something like `params: { <model>: { ... } }`? Or simply `<model>_params` or `<model>_url_options` ?
I would agree indeed for a change towards the latter but we still need a way to characterize these options by model as we might want to pass different options for each of the involved models in `affair_of`.

Maybe `url_options: { <model>: { ... }, <other_model>: { ... } }` ?